config/config.go

```diff
@@ -47,6 +48,21 @@ func New() *Config {
 	return c
 }
 
+// NewFromKiteKey parses the given kite key file and gives a new Config value.
+func NewFromKiteKey(file string) (*Config, error) {
+	key, err := kitekey.ParseFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	var c Config
```

Comment:
any reason this is not `*Config`...don't usually see non pointer declaration and return with `&`